relay/kusama/src/lib.rs

```diff
@@ -2006,6 +1944,11 @@ pub mod migrations {
 			<Runtime as frame_system::Config>::DbWeight,
 		>,
 		CancelAuctions,
+		// Remove `identity` pallet on-chain storage. `identity-migrator` is stateless.
+		frame_support::migrations::RemovePallet<
```

Comment:
I guess this is only required for pallets that also had storage items? (thinking if we need it for `identity_migrator` too)

Reply:
AFAIK yes, but yeah perhaps @liamaharon or @ggwpez could confirm.

Reply:
I just checked and the `identity_migrator` has no storage items

Reply:
> I just checked and the `identity_migrator` has no storage items

There should still be the `StorageVersion` that should be removed.